src/main/java/com/softserveinc/ita/pageobjects/BasePage.java

```diff
@@ -0,0 +1,49 @@
+package com.softserveinc.ita.pageobjects;
+
+import static com.codeborne.selenide.Selenide.$x;
+
+public abstract class BasePage {
+
+    private String menuButtonSelector = "//button[@class = 'header__button']";
+    private String catalogButtonSelector = "//button[contains(@class, 'menu__toggle')]";
+    private String searchBarSelector = "//input[@name = 'search']";
+    private String searchButtonSelector = "//button[contains(@class, 'search-form__submit')]";
+    private String logInButtonSelector = "//rz-user[@class='header-actions__component']";
+    private String basketButtonSelector = "//rz-cart[@class = 'header-actions__component']";
+
+    public BasePage openMenu() {
+        $x(menuButtonSelector).click();
```

Comment:
- move all selectors from class variables into methods, if they are used only once in one method;
 - in future let's store in class variables SelenideElement instead of selectors, and make they final